src/modules/fetchComment.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable   */
```

Comment:
- [ ] Kindly do not disable linters as they help us write better and cleaner code